lib/fakeroot.py

```diff
@@ -221,6 +224,22 @@
      "cmds": ["apt", "apt-get", "dpkg"],
      "each": ["fakeroot"] },
 
+   # Fedora notes:
+   #
+   # 1. The minimum supported version was chosen somewhat arbitrarily based on the
+   #    release versions available for testing (what was on Dockerhub).
```

Comment:
I'm going to tidy this comment by spelling Docker Hub correctly. You would be correct to point out it's a trivial change; the purpose is to make searches for "Docker Hub" find this spot.

```suggestion
   #    release versions available for testing (what was on Docker Hub).
```